core/src/main/java/io/substrait/expression/Expression.java

```diff
@@ -684,41 +717,6 @@ public <R, E extends Throwable> R accept(ExpressionVisitor<R, E> visitor) throws
     }
   }
 
-  @Value.Immutable
-  abstract static class Window implements Expression {
-    @Nullable
-    public abstract Aggregate.Measure aggregateFunction();
-
-    @Nullable
-    public abstract WindowFunction windowFunction();
```

Comment:
Instead of having separate handling for window calls with window functions, and window calls with aggregate functions, this PR updates the extension handling to treat all aggregate functions as valid window functions and a such removes the need for this distinction.